docs/configuration/backends/kubernetes.md

```diff
@@ -118,6 +118,7 @@ The following general annotations are applicable on the Ingress object:
     Replaces each matched Ingress path with the specified one, and adds the old path to the `X-Replaced-Path` header.
 - `ingress.kubernetes.io/whitelist-source-range: "1.2.3.0/24, fe80::/16"`
     A comma-separated list of IP ranges permitted for access. all source IPs are permitted if the list is empty or a single range is ill-formatted.
+-  `ingress.kubernetes.io/app-root: /index.html`  Redirects all requests for / to the defined path. Non-root paths will not be affected by this annotation and handled normally. This annotation may not be combined with the ReplacePath rule type any other annotation leveraging that rule type. Trying to do so leads to an error and the corresponding Ingress object being ignored.
```

Comment:
There's an article missing: Please replace

_This annotation may not be combined with the ReplacePath rule type any other annotation leveraging that rule type._

by (**emphasis** by me):

_This annotation may not be combined with the ReplacePath rule type **or** any other annotation leveraging that rule type._